ckanext/subscribe/templates/group/snippets/info.html

```diff
@@ -15,7 +15,7 @@
   {% if current_user.is_authenticated %}
     {{ super() }}
   {% else %}
-    {% asset 'subscribe/css/subscribe' %}
+    {% asset 'subscribe/subscribe-css' %}
```

Comment:
it should be `{% asset 'subscribe-css' %}`, or we need to keep `subscribe/subscribe-css:` , when we def. it